staging/src/github.com/kubeedge/mapper-framework/_template/mapper/go.mod

```
@@ -37,3 +38,8 @@ require (
 	golang.org/x/text v0.13.0 // indirect
 	google.golang.org/genproto v0.0.0-20230306155012-7f2fa6fef1f4 // indirect
 )
+
+replace (
+	github.com/kubeedge/kubeedge => ../../../../../../../../kubeedge
+	github.com/kubeedge/mapper-framework => ../../../mapper-framework
```

```suggestion
	github.com/kubeedge/kubeedge => ../../../../../../kubeedge
	github.com/kubeedge/mapper-framework => ../mapper-framework
```

Considering the e2e test of mapper-framework, the directory where the generated mapper project is located should be used as the standard for dependent applications. Otherwise, the directory actually referenced by the generated mapper project is incorrect.

---

> Considering the e2e test of mapper-framework, the directory where the generated mapper project is located should be used as the standard for dependent applications. Otherwise, the directory actually referenced by the generated mapper project is incorrect.

ok